bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/DropDown.java

```diff
@@ -34,4 +34,7 @@ public interface DropDown extends Field, OptionsConstraint, ContainerConstraint
      */
     @JsonIgnore
     Boolean isMultiSelect();
+
+    @JsonIgnore
+    Boolean isEnableSuggestions();
```

Comment:
Please add java doc. Since there is a change in interface, please bump the minor version in package-info file